packages/theme-chalk/src/button.scss

```diff
@@ -40,7 +40,7 @@ $button-icon-span-gap: map.merge(
   vertical-align: middle;
   -webkit-appearance: none;
 
-  &:not(.is-text, .is-link, .#{$namespace}-button--text) {
+  &:not(.is-text):not(.is-link):not(.#{$namespace}-button--text) {
```

Comment:
I think this originally means 

```css
&:not(.is-text), &:not(.is-link), &:not(.#{$namespace}-button--text) {}
```

Reply:
`&:not(a, b)` same with `&:not(a):not(b)` , but the `&:not(a), &:not(b)` is completely different.

&:not(a), // just exclude a, but include b
&:not(b) // just exclude b, but include a

Reply:
Right, my bad.